dockstore-integration-testing/src/test/java/io/dockstore/client/cli/BasicIT.java

```diff
@@ -197,7 +197,7 @@ public void testRegistrationWithNonLowerCase() {
 
         // refresh a specific workflow
         Workflow workflow = workflowsApi
-                .getWorkflowByPath(SourceControl.GITHUB.toString() + "/DockstoreTestUser/dockstore-whalesay-wdl", "", BIOWORKFLOW);
+                .getWorkflowByPath(SourceControl.GITHUB.toString() + "/DockstoreTestUser/dockstore-whalesay-wdl", BIOWORKFLOW, "");
```

Comment:
Why did the order of params swap, and does the swap break anything? I think it will be OK, but just making sure.

Reply:
Not sure, the order switch may be slightly inconvenient for the corresponding CLI fix, but since we're adding a required parameter I think changes are unavoidable. But already compiled clients should be sending over query parameters which don't have an order.